indico/htdocs/js/indico/modules/contributions/common.js

```diff
@@ -193,6 +193,17 @@
         });
     }
 
+    function setupFilterDialog() {
+        $('#filter-button').ajaxDialog({
+            title: $T('Contribution list configuration'),
+            onClose: function(data) {
+                if (!data) { return }
```

Comment:
I know it's tempting, but if we start it with empty returns, someone will do it for returns with values, and so on